utilities/rev.py

```diff
@@ -0,0 +1,17 @@
+import rev
+
+
+def configure_spark_ephemeral(motor: rev.SparkBase, config: rev.SparkBaseConfig):
+    motor.configure(
+        config,
+        rev.SparkBase.ResetMode.kNoResetSafeParameters,
+        rev.SparkBase.PersistMode.kNoPersistParameters,
+    )
+
+
+def configure_spark_persistent(motor: rev.SparkBase, config: rev.SparkBaseConfig):
```

Comment:
I think we should be explicit that we're resetting all the otherwise unset configs when we're burning the config to flash, rather than having this helper function. Might be surprising to a reader otherwise.

Reply:
amended to fix this